### PR TITLE
add Navbar

### PR DESCRIPTION
**Pull request by TheSwerik**

### CODE REVIEW

File: Shared/Navbar.razor.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    public partial class Navbar
+    {
+        private Guid _user = new Guid();
+        [Inject] private NavigationManager MyNavigationManager { get; set; }
+        [Inject] private UserService UserService { get; set; }
+
+        protected override void OnInitialized()
+        {
+            base.OnInitialized();
+            Console.WriteLine(MyNavigationManager.Uri);
```
Comment (DennisGeb) [inline Shared/Navbar.razor.cs:16]: is this supposed to be here ?

File: wwwroot/index.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
 	<base href="/"/>
 	<link href="css/bootstrap/bootstrap.min.css" rel="stylesheet"/>
 	<link href="css/app.css" rel="stylesheet"/>
+	<link href="css/Navbar.css" rel="stylesheet"/>
```
Comment (DennisGeb) [inline wwwroot/index.html:11]: are u sure these are needed ?

**Review by DennisGeb — CHANGES_REQUESTED** [review]

File: Shared/Navbar.razor [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                }
+            </li>
+            <li class="nav-item @(false ? "active" : "")">
+                @if (UserService.CurrentUser.Id != Guid.Empty)
+                {
+                    <a class="nav-link" href="user/@UserService.CurrentUser.Id">@UserService.CurrentUser.Username</a>
+                }
+            </li>
+            <li class="nav-item @(isLogin() ? "active" : "")">
+                <a class="nav-link" href="/login">Login</a>
```
[the remark this answers is not in the document]
Comment (TheSwerik) [reply Shared/Navbar.razor:24]: already fixed in eed1941

File: wwwroot/index.html (hunk 2 above)
Comment (TheSwerik) [reply wwwroot/index.html:11]: No it was to learn how to do it and to give an example

app.css and bootstrap.min.css are **required** but Navbar isn't, i think I even deleted it in login-register branch

File: Shared/Navbar.razor.cs (hunk 1 above)
Comment (TheSwerik) [reply Shared/Navbar.razor.cs:16]: No but already removed in login-register branch

**Review by DennisGeb — APPROVED** [review]

**Pull request merged**